src/bindings/python/__init__.py

```diff
@@ -129,7 +129,7 @@ class Object:
     ], new_base = Units)
 import libcellml
 import libcellml.specificationrule
-convert(specificationrule, 'SpecificationRule', [
+convert(specificationrule, 'ReferenceRule', [
```

Comment:
Shouldn't this be `convert(referencerule, 'ReferenceRule', [`.

Reply:
Done